llama_poe/data/apps.md

```diff
@@ -0,0 +1,22 @@
+# Integrations into LLM Applications
+
+LlamaIndex modules provide plug and play data loaders, data structures, and query interfaces. They can be used in your downstream LLM Application. Some of these applications are described below.
+
+### Chatbots
+
+Chatbots are an incredibly popular use case for LLM's. LlamaIndex gives you the tools to build Knowledge-augmented chatbots and agents.
+
+Relevant Resources:
+- [Building a Chatbot](/guides/tutorials/building_a_chatbot.md)
```

Comment:
This link won't work in this repo, will it?

Reply:
These are just example data, not meant as actual guide.

Reply:
Oh I see now. It's pretty relevant information :)